data/data/config.tf

```diff
@@ -65,6 +65,11 @@ EOF
 
 // This variable is generated by OpenShift internally. Do not modify
 variable "cluster_id" {
-  type        = "string"
-  description = "(internal) The OpenShift cluster id."
+  type = "string"
+
+  description = <<EOF
+(internal) The OpenShift cluster id.
+
+This cluster id must be of max length 32 and must have only alphabumeric or hyphen characters.
```

Comment:
The max length is 27, not 32.

Reply:
done https://github.com/openshift/installer/pull/1280/commits/b156efaf37b835972ccd68b9ec07279f05cdad24